client/src/charts/legends/StandardLegend.js

```diff
@@ -31,16 +31,18 @@ export const StandardLegend = ({
           }}
         />
       </div>
-      {Controls && (
-        <div
-          style={{
-            borderTop: `1px dashed ${window.infobase_color_constants.tertiaryColor}`,
-            padding: "10px 0px 10px 5px",
-          }}
-        >
-          {Controls}
-        </div>
-      )}
+      {Controls &&
+        _.map(Controls, (Control, key) => (
+          <div
+            key={key}
+            style={{
+              borderTop: `1px dashed ${window.infobase_color_constants.tertiaryColor}`,
+              padding: "10px 0px 10px 5px",
+            }}
+          >
+            {Control}
+          </div>
+        ))}
```

Comment:
So this was when I had the toggle button inside the legend. I got rid of the button inside legend in favor of a different location, and now a radio button. But just left this change in case we want multiple controls in the legend. Then it'd be better for aesthetics to have dashed border between each control.

I could remove this change if you don't like it

Reply:
Sorry, that thumbs up was supposed to mean "yes, please remove this change, thanks". Should have just clearly responded, haha.